common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1930,6 +1931,15 @@ object CelebornConf extends Logging {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("1m")
 
+  val WORKER_PARTITION_SPLIT_MAX_SIZE: ConfigEntry[Long] =
+    buildConf("celeborn.worker.shuffle.partitionSplit.max")
+      .withAlternative("celeborn.shuffle.partitionSplit.max")
```

Comment:
I think we don't need ```withAlternative``` here, it's only used for backward compatibility.

Reply:
`withAlternative` is the alias, we don't need it for newly added configuration

Reply:
ok, i remove it